tests/spread/general/license-valid/task.yaml

```diff
@@ -26,4 +26,4 @@ execute: |
 
   snapcraft  
 
-  cat meta/snap.yaml | MATCH -v "license: MIT"
+  MATCH -v "license: MIT" < meta/snap.yaml
```

Comment:
```
Snapped my-snap-name_0.1_amd64.snap
+ MATCH -v 'license: MIT'
/bin/bash: line 33: meta/snap.yaml: No such file or directory
```
You uncovered something here, the path should be prime/meta/snap.yaml

Reply:
well spotted, thanks!